docs/fuel-101/understanding-the-bridge.mdx

```diff
@@ -0,0 +1,76 @@
+- the `sendMessage` function in the [`FuelMessagePortal`](https://github.com/FuelLabs/fuel-bridge/tree/main/packages/solidity-contracts/contracts/fuelchain/FuelMessagePortal.sol) Solidity contract
+(if not just ETH:)
+- the `process_message` function in the [Bridge Fundible Token](https://github.com/FuelLabs/fuel-bridge/tree/main/packages/fungible-token/bridge-fungible-token/src/main.sw) Sway contract
+
+## How does a withdrawl work?
+
+To withdraw, or bridge something from Fuel to Ethereum:
+
+1. A user initiates a withdrawl of anything on the bridge contract on Fuel.
+2. In the transaction, the asset/data on the Fuel network is burned, and a message with the withdrawl information is included in the receipts.
+3. The transaction is confirmed and included in a block on Fuel.
+4. The block is included in an epoch on the Fuel network. An epoch a group of Fuel blocks, packed together and identified by the last block of the epoch.
+5. Once the epoch is reached, the last block hash in the epoch is committed to the L1 blockchain in the [Chain State contract](https://github.com/FuelLabs/fuel-bridge/tree/main/packages/solidity-contracts/contracts/fuelchain/FuelChainState.sol). This allows anyone to prove the inclusion of all the blocks in the epoch with the Merkle root of the Fuel `rootBlock` (the last block of a Fuel epoch).
+6. The transaction is confirmed and included in a block on Ethereum.
+7. A grace period must pass before this data is considered finalized and usable to prove the withdrawl was initiated.
+8. The user must finalize the withdrawl on the bridge contract on Ethereum by calling the `relayMessage` function with the commmited block and merkle proof data.
+This function will then trigger `finalizeWithdrawal` function in the `FuelERC20Gateway` contract on Ethereum with the withdrawl information.
+9. Once this final transaction confirmed and included in a block on Ethereum, the assets will be available in the destination wallet.
+
+You can follow the actual implementation via:
+
+- the `withdraw` function in the [Bridge Fundible Token](https://github.com/FuelLabs/fuel-bridge/tree/main/packages/fungible-token/bridge-fungible-token/src/main.sw) Sway contract
+- the `commit` and `finalized` functions in the [`FuelChainState`](../packages/solidity-contracts/contracts/fuelchain/FuelChainState.sol) Solidity contract
+- the `relayMessage` function in the [`FuelMessagePortal`](https://github.com/FuelLabs/fuel-bridge/tree/main/packages/solidity-contracts/contracts/fuelchain/FuelMessagePortal.sol) Solidity contract
+- the `finalizeWithdrawal` function in the [`FuelERC20Gateway`](https://github.com/FuelLabs/fuel-bridge/tree/main/packages/solidity-contracts/contracts/messaging/gateway/FuelERC20Gateway.sol) Solidity contract
+
+## How can I use the bridge?
+
+You can try out the Fuel bridge by going to the [bridge UI](https://next-app.fuel.network/bridge), connecting your wallet, and following the instructions on the page.
```

Comment:
```suggestion
You can try out the Fuel bridge by going to the [bridge UI](https://app.fuel.network/bridge), connecting your wallet, and following the instructions on the page.
```
??

Reply:
is there a difference between these urls?